<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerPopover</class>
 <widget class="QWidget" name="PowerPopover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="tTitleLabel" name="titleLabel">
     <property name="text">
      <string>Power Options</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>What do you want to do now?</string>
     </property>
     <property name="margin">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="powerOffButton">
     <property name="text">
      <string>Power Off</string>
     </property>
     <property name="icon">
      <iconset theme="system-shutdown">
       <normaloff>../../thedesk/desktop/session</normaloff>../../thedesk/desktop/session</iconset>
     </property>
     <property name="description">
      <string>Turns off your computer.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="rebootButton">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="text">
      <string>Reboot</string>
     </property>
     <property name="icon">
      <iconset theme="system-reboot">
       <normaloff>../../thedesk/desktop/session</normaloff>../../thedesk/desktop/session</iconset>
     </property>
     <property name="description">
      <string>Turns off your computer and turns it on again</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tTitleLabel</class>
   <extends>QLabel</extends>
   <header location="global">ttitlelabel.h</header>
   <slots>
    <signal>backButtonClicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
